packages/grid/_modules_/grid/hooks/core/preProcessing/gridPreProcessingApi.ts

```diff
@@ -4,16 +4,15 @@ export enum GridPreProcessingGroup {
   hydrateColumns = 'hydrateColumns',
 }
 
-export interface GridPreProcessingApi {
+export interface GridPreProcessingPrivateApi {
   /**
    * Register a pre-processor and emit an event to notify the agents to re-apply the pre-processors.
    * @param {GridPreProcessingGroup} group The name of the group to bind this pre-processor to.
    * @param {number} id An unique and static identifier of the pre-processor.
    * @param {PreProcessorCallback} callback The pre-processor to register.
    * @returns {() => void} A function to unregister the pre-processor.
-   * @ignore - do not document.
```

Comment:
We don't need to ignore them or prefix them by `unstable_` anymore since they are not on `GridApi`